app/controllers/katello/api/v2/subscriptions_controller.rb

```diff
@@ -81,24 +81,18 @@ def create
       object = @system || @activation_key || @distributor
 
       if params[:subscriptions]
-        params[:subscriptions].each do |sub|
-          subscription = Pool.find(sub[:id])
-          object.subscribe(subscription.cp_id, subscription[:quantity])
+        params[:subscriptions].each do |sub_params|
+          subscription = Pool.find(sub_params[:id])
+          object.subscribe(subscription.cp_id, sub_params[:quantity])
+          subscription.import_data
```

Comment:
You can always pick a resource that you know is dynflowed (e.g. product creation) and try and follow that. There are also a number of recordings that cover Dynflow and conversion: https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&es_th=1&ie=UTF-8#q=dynflow&tbm=vid

I think those are two good places to start and then we can help fill in the gaps.